package/firmware/fman-ucode/Makefile

```diff
@@ -9,13 +9,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
 PKG_NAME:=fman-ucode
-PKG_VERSION:=fsl-sdk-v2.0
+PKG_VERSION:=2017.07
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_MIRROR_HASH:=209c60e4efaa3ce9c9a954266a7210fa0047a2de47ce00a412aa0318d94d0d88
-PKG_SOURCE_URL:=https://github.com/fsl-jyt/fm-ucode.git
-PKG_SOURCE_VERSION:=b19c645821941493fbef32e616b5a16404259976
+PKG_MIRROR_HASH:=80d8d2454063b7f730ef49d7a44be8f8870711ee5109c53a0881b62026037a37
+PKG_SOURCE_URL:=https://github.com/qoriq-open-source/fm-ucode.git
+PKG_SOURCE_VERSION:=c275e91392e2adab1ed22f3867b8269ca3c54014
 
 PKG_BUILD_DIR=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 
```

Comment:
Oh, you're right... These're binaries developed by Freescale/NXP.
Let me remove the error PKG_LICENSE.

May I know whether PKG_LICENSE is required in Makefile? Could I just ignore it.
fman-ucode is neither GPL or BSD, it's just Freescale-Binary-EULA.